### DISCUSSION TITLE
Targa support?

### DISCUSSION THREAD

**Discussion by cycyx** [deckerst/aves#1126] [Q&A]
I still haven't found (and have not thoroughly searched to be honest) an app able to display tga files in Android.

Is this something Aves could do?

**Comment by deckerst** [comment]
Unlikely. Aves supports mainstream image formats: the ones shipped with Android, plus TIFF which is important in the scientific world. For other formats, there are already desktop applications that try to support everything.

**Comment by cycyx** [comment]
Once again, thank you for all your answers and for Aves! 👍
